doc/asciidoc/consumer/index.adoc

```diff
@@ -483,7 +483,22 @@ Config Options
 | errors.deadletterqueue.topic.replication.factor | 3/1 | replication factor, need to set to 1 for single partition, default:3
 |===
 
-In Kafka Connect you can just configure the above properties.
+In Kafka Connect in addition to the above properties you need to define kafka broker connection properties:
+
+|===
+| Name | mandatory | Description
+
+| kafka.bootstrap.servers | true | It's the Kafka Broker url,
+this because the Kafka Connect Framework provides an out-of-the-box support
```

Comment:
probably move the long text behind the table
and also link to the JIRA issue and point out this is a shortcoming in the connect framework

Reply:
done